lucetc/tests/wasm.rs

```diff
@@ -2,15 +2,35 @@ use lucet_module::bindings::Bindings;
 use std::collections::HashMap;
 use std::path::PathBuf;
 
-pub fn load_wat_module(name: &str) -> Vec<u8> {
+struct LoadOptions {
+    write_debug_names: bool,
+}
+
+impl Default for LoadOptions {
+    fn default() -> Self {
+        LoadOptions {
+            write_debug_names: false,
+        }
+    }
+}
+
+fn load_wat_module(name: &str, options: LoadOptions) -> Vec<u8> {
```

Comment:
For our name section tests, we need to compile _with_ the names section included. The `wat2wasm` function does [not do this by default](https://github.com/pepyakin/wabt-rs/blob/master/src/lib.rs#L511), so I essentially vendored part of that structure here, since it's not exposed by `wabt-rs`.

This nice thing about this is that we can continue to share code for loading wat modules, and opt into debug information when we need to write tests about that. :slightly_smiling_face: